projects/core/src/util/resolveObject.ts

```diff
@@ -0,0 +1,40 @@
+import { forkJoin, Observable, of } from 'rxjs';
+import { map, mergeMap, reduce } from 'rxjs/operators';
+
+export function resolveValuesBy<O extends { [key: string]: IN }, IN, OUT>(
```

Comment:
I cannot find usages of those functions anymore. @Pucek9 would you mind if we remove them?

Reply:
No need to ask, it's obsolete and was not exported as well. I've dropoed it.